py/desispec/fiberflat_vs_humidity.py

```diff
@@ -159,7 +159,9 @@ def compute_humidity_corrected_fiberflat(calib_fiberflat, mean_fiberflat_vs_humi
     if frame is not None :
         log.info("using frame to fit for the best fit current humidity")
         ivar = frame.ivar*(frame.mask==0)
-        badfibermask = get_skysub_fiberbitmask_val()
+        camera = frame.meta["CAMERA"].lower()
```

Comment:
Earlier you check for both frame.meta and that CAMERA is in frame.meta. Should you do that here as well?

Reply:
yes